File: HelloWorld/Mang2chieu/bai5.cpp
#include<iostream>
using namespace std;
void input(int a[][50], int &m, int &n, int t)
{
    cout << "Nhap vao mang 2 chieu " << t << " :" << endl;
    for (int i = 0; i < m; i++)
    {
        for (int j = 0; j < n; j++)
        {
            cin >> a[i][j];
        }
    }
}
int main()
{
    int m, n, p, q, a[50][50], b[50][50], c[50][50];
    cout << "Nhap so dong mang 2 chieu 1 : ";
    cin >> m;
    cout << "Nhap so cot mang 2 chieu 1 : ";
    cin >> n;
    input(a, m, n, 1);
    cout << "Nhap so dong mang 2 chieu 2 : ";
    cin >> p;
    cout << "Nhap so cot mang 2 chieu 2 : ";
    cin >> q;
    input(b, p, q, 2);
    if (n == p)
    {
        for (int i = 0; i < m; i++)
        {
            for (int j = 0; j < q; j++)
            {
                for (int t = 0; t < n; t++)
                {
                    c[i][j] += (a[i][t] * b[t][j]);
                }
            }
        }
        cout << "Mang sau khi nhan duoc la : " << endl;
        for (int i = 0; i < m; i++)
        {   
            for (int j = 0; j < q; j++)
            {
                cout << c[i][j] << " ";
            }
            cout << endl;
        }
    }
    else
    {
        cout << "Khong the nhan 2 mang";
    }
}

File: Java/Lab1/bai5.java
import java.util.Scanner;
import java.lang.Math;
import java.util.Arrays;
public class bai5 {
    public static void main(String[] args) {
        int m, n;
        Scanner input = new Scanner(System.in);
        System.out.print("Nhap vao chieu dai mang A: ");
        n = input.nextInt();
        int[] a = new int[n];
        System.out.print("Nhap vao mang A: ");
        for (int i = 0; i < n; i++)
        {
            a[i] = input.nextInt();
        }
        System.out.print("Nhap vao chieu dai mang B: ");
        m = input.nextInt();
        int[] b = new int[m];
        for (int i = 0; i < m; i++)
        {
            b[i] = (int)(Math.random() * 100);
        }
        System.out.print("Mang B sau khi tao: ");
        for (int i = 0; i < m; i++)
        {
            System.out.print(b[i] + " ");
        }
        int[] c = Arrays.copyOf(a, n);
        System.arraycopy(b, m - 3, c, 0, 3);
        for (int i = 0; i < n - 1; i++)
        {
            for (int j = i + 1; j < n; j++)
            {
                if (c[i] >= c[j])
                {
                    int temp = c[i];
                    c[i] = c[j];
                    c[j] = temp;
                }
            }
        }
        System.out.print("\nMang C sau khi tao: ");
        for (int  i = 0; i < n; i++)
        {
            System.out.print(c[i] + " ");
        }
    }
}
